### DISCUSSION TITLE
機器人公司出高價買「和善的臉」

### DISCUSSION THREAD

**Discussion by qeeeee39** [1101-GS3073-C/Python-Community-GS3073-C#224] [General]
Promobot 是機器人新創，2015 年在俄羅斯彼爾姆成立，自稱是北歐和東歐最大的「服務機器人」製造商。他開發機器人，並且是機電一體化（從軟體到硬體都是自己在設計開發）。目前提供了三種機器人產品：服務機器人、仿人機器人、量體溫機器人。

在仿人機器人這塊，Promobot開始向「長相和藹的人」購買他的肖像權和聲音，試圖用創造逼真的機器人。不過目前還不知道是不是真的只用在機器人的外觀和聲音之上。由於Promobot開價很高（一張人臉加聲音20萬美金），很多人急著把臉拿去賣，多到公司急喊停收。

我認為這個買臉的動作很爭議，在這個資訊時代臉好像也是需要被保護的個人資訊了，但Promobot用最公開的方式買臉好像又沒有什麼問題。

https://technews.tw/2021/12/14/promobot/

**Comment by ghost** [comment]
20萬美金....如果是我我好想飛去俄羅斯賣臉XD
